go.mod

```diff
@@ -104,3 +104,5 @@ replace (
 	k8s.io/utils v0.0.0 => k8s.io/utils v0.21.4
 	sigs.k8s.io/apiserver-network-proxy/konnectivity-client => sigs.k8s.io/apiserver-network-proxy/konnectivity-client v0.0.22
 )
+
+replace google.golang.org/grpc => google.golang.org/grpc v1.26.0
```

Comment:
we don't need this line? there is already a `google.golang.org/grpc v1.38.0` in line 39

Reply:
This fixed grpc version are not compatible, see https://github.com/asim/go-micro/issues/2091

Reply:
How will it going if we move this line to L39?

Reply:
> How will it going if we move this line to L39?

```
github.com/kubeedge/kubeedge/edge/pkg/metamanager/metaserver/kubernetes/storage/sqlite/imitator imports
        k8s.io/apiserver/pkg/storage/etcd3 tested by
        k8s.io/apiserver/pkg/storage/etcd3.test imports
        go.etcd.io/etcd/integration imports
        go.etcd.io/etcd/proxy/grpcproxy imports
        google.golang.org/grpc/naming: module google.golang.org/grpc@latest found (v1.43.0), but does not contain package google.golang.org/grpc/naming
```

Reply:
I have compressed the commits that would be easier to review. :）